.github/workflows/ci.yaml

```diff
@@ -43,14 +40,6 @@ jobs:
         create-args: >-
           python=${{ matrix.python-version }}
 
-    - name: License OpenEye
-      shell: bash -l {0}
-      run: |
-        echo "${SECRET_OE_LICENSE}" > ${OE_LICENSE}
-        python -c "from openeye import oechem; assert oechem.OEChemIsLicensed()"
-      env:
-        SECRET_OE_LICENSE: ${{ secrets.OE_LICENSE }}
-
```

Comment:
This was not needed for direct API access or exclusive functionality, it was just to get charge assignment running faster. This can be re-added at any time provided the org's license is managed, or NAGL could be used as a replacement for quicker charge assignment in tests